apps/serverless/src/app.ts

```diff
@@ -0,0 +1,11 @@
+import { IncomingMessage, ServerResponse } from "http";
```

Comment:
The `http` dependency is missing from `package.json`.

Reply:
fixed